src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  transition-duration: 1s;
```

Comment:
It's good that you are using CSS transitions, but it would be helpful to specify which properties you want to transition. If you want to apply the transition to all properties, you can use 'all', like this: 'transition: all 1s;'. If you want to apply the transition to specific properties, you can list them, like this: 'transition: background-color 1s, color 1s;'. It's a good practice to specify the properties to avoid unnecessary performance issues.